client/src/components/item/ItemEdit.tsx

```diff
@@ -94,35 +94,35 @@ class ItemEdit extends Component<ItemEditProps, ItemEditState> {
         }
     }
 
-    handleInputChangeDropdown(name: string, dropdownProps: DropdownProps): void {
+    public handleInputChangeDropdown(name: string, dropdownProps: DropdownProps): void {
         const value = dropdownProps.value;
     }
 
-    handleInputChange(event: ChangeEvent<HTMLInputElement>): void {
+    public handleInputChange(event: ChangeEvent<HTMLInputElement>): void {
         const target = event.target;
-        const value = target.type === 'checkbox' ? target.checked : target.value;
+        const value = target.type === "checkbox" ? target.checked : target.value;
         const name = target.name;
 
         this.setState({
             [name]: value
         });
     }
 
-    render() {
-        const categories = ['Extension Cords', 'Laptops', 'Microcontrollers', 'Robotics'];
+    public render() {
+        const categories = ["Extension Cords", "Laptops", "Microcontrollers", "Robotics"];
```

Comment:
same here for config. probably do a single PR purging constants

Reply:
Categories will be pulled from the server in the final version of this screen; this was just a placeholder